public/app/features/dashboard/dashgrid/PanelStateWrapper.tsx

```diff
@@ -679,7 +681,7 @@ export class PanelStateWrapper extends PureComponent<Props, State> {
           dragClassCancel="grid-drag-cancel"
           padding={padding}
           hoverHeaderOffset={hoverHeaderOffset}
-          hoverHeader={title ? false : true}
+          hoverHeader={this.hasOverlayHeader()}
```

Comment:
Why not use `hasHoverHeader` again?

Reply:
Ah, my mistake I added the comet variable to not have to call the function twice , must have missed updating this